### DISCUSSION TITLE
Cut Settings Units of Measure are Different

### DISCUSSION THREAD

**Discussion by DaveSkowron** [winder/Universal-G-Code-Sender#2500] [Help] [answered]
I'm on v2.1.6 / Ubuntu. I have the units in the visualizer set to inches. The tool sizes, material thickness and Cut Settings window are in mm. The menu item "Machine>Toggle Units" only toggles the visualizer. How do I get the rest of the program's features to use inches?

**Comment by breiler** [comment, marked as answer]
I have not yet implemented support for inches in the Designer tool.

**Reply by DaveSkowron** [reply]
Thanks!
